.github/workflows/release.yml

```diff
@@ -2,7 +2,7 @@ name: Release
 on:
   push:
     tags:
-      - 'v[0-9]+.[0-9]+.[0-9]+'
+      - 'v[0-9]{1-4}+.[0-9]{1-4}+.[0-9]{1-4}+'
```

Comment:
I'm a bit confused if how this interacts with the plus, and why not just use `\d` instead of `[0-9]`?